src/script/page/MainContent/panels/preferences/DevicesPreferences/components/DetailedDevice.tsx

```diff
@@ -29,7 +29,6 @@ import {ProteusDeviceDetails} from './ProteusDeviceDetails';
 export interface DeviceProps {
   device: ClientEntity;
   fingerprint: string;
-  showVerificationStatus?: boolean;
```

Comment:
The API could be simplified: if the `isProteusVerified` is undefined, then we don't show any verification status